src/runtime/config/configuration-remote.toml.in

```diff
@@ -0,0 +1,310 @@
+# Copyright (c) 2017-2019 Intel Corporation
+# Copyright (c) 2023 IBM Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# XXX: WARNING: this file is auto-generated.
```

Comment:
I understand that we generate files from this template, but I find it weird that we have "auto-generated" in the template ;-)